packages/squiggle-lang/__tests__/Distributions/Symbolic_test.res

```diff
@@ -0,0 +1,177 @@
+open Jest
+open Expect
+
+let fnImage = (theFn, inps) => Js.Array.map(theFn, inps)
+
+let env: DistributionOperation.env = {
+  sampleCount: 100,
+  xyPointLength: 100,
+}
+
+let mkNormal = (mean, stdev) => GenericDist_Types.Symbolic(#Normal({mean: mean, stdev: stdev}))
+let {toFloat, toDist, toString, toError, fmap} = module(DistributionOperation.Output)
+let run = DistributionOperation.run(~env)
+let outputMap = fmap(~env)
+let toExtFloat: option<float> => float = E.O.toExt(
+  "Should be impossible to reach (This error is in test file)",
+)
+let toExtDist: option<GenericDist_Types.genericDist> => GenericDist_Types.genericDist = E.O.toExt(
+  "Should be impossible to reach (This error is in a test file)", 
```

Comment:
String repeated twice, plus in another file. We probably could use that Test_Helpers file now.